### PR TITLE
Restore a top-level header

### PR DESCRIPTION
**Pull request by willbarton**
#120 left our README a bit headless.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# CFPB development team
 
 > A guide for the development team at the [CFPB](http://cfpb.github.io/).
```
Comment (Scotchester) [inline README.md]: While we're here, let's pull this out of the blockquote styling and make this link HTTPS.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# CFPB development team
```
Comment (Scotchester) [inline README.md]: The title should reflect the repo, really. "CFPB Development Guidelines"?

**Comment by ascott1** [comment]
👍

**Comment by Scotchester** [comment]
:+1:

File: README.md [hunk 3]
```diff
@@ -1,10 +1,11 @@
+# CFPB development guidelines
 
-> A guide for the development team at the [CFPB](http://cfpb.github.io/).
+A guide for the development team at the [CFPB](https://cfpb.github.io/).
 
 ## Standards and styles
 
-> All code in any code-base should look like a single person typed it, no matter how many people contributed.
-> - [idiomatic.js](https://github.com/rwaldron/idiomatic.js/#all-code-in-any-code-base-should-look-like-a-single-person-typed-it-no-matter-how-many-people-contributed)
+All code in any code-base should look like a single person typed it, no matter how many people contributed.
```
Comment (Scotchester) [inline README.md]: Actually, I retract the 👍 . This doesn't render correctly anymore with the removal of _this_ blockquote styling, which is actually a quote.

**Comment by Scotchester** [comment]
@willbarton I can't figure out why this is still saying it's conflicted. I resolved the conflicts using their web interface.

**Pull request merged**